bin/server.js

```diff
@@ -5,17 +5,6 @@
 var fs = require('fs');
 var http = require('http');
 var path = require('path');
-var spawn = require('child_process').spawn;
```

Comment:
taken out and made an npm script instead, see https://github.com/nodebox/opentype.js/pull/131/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R32 below